### ISSUE TITLE
Dataset Viewer issue for eli5

### ISSUE THREAD

**Issue by surya-narayanan** [huggingface/datasets-server#1154]
### Link

https://huggingface.co/datasets/eli5

### Description

The dataset viewer is not working for dataset eli5.

Error details:

```
Error code:   UnexpectedError

```

**Comment by albertvillanova** [comment]
Thanks for reporting, @surya-narayanan.

Please note this is a known issue and we are working on it: https://huggingface.co/datasets/eli5/discussions/2
- Let's continue the discussion there

Also note that first we should address another issue: https://huggingface.co/datasets/eli5/discussions/3
